Makefile

```diff
@@ -438,6 +438,24 @@ $(BUILD_DIR)/assets/%.o: assets/%.c
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $<
 	$(OBJCOPY) -O binary $@ $@.bin
 
+# Temporary hacks to handle asset differences
+ifeq ($(VERSION),gc-eu-mq)
+$(BUILD_DIR)/assets/text/message_data.enc.h:
+	$(PYTHON) tools/msgdis.py --oot-version $(VERSION) $(BUILD_DIR)/assets/text/message_data.h
```

Comment:
this would be less weird to change with part of #1491

Reply:
Maybe, although I think we'd still need the hack here to avoid clobbering the debug ROM assets